### ISSUE TITLE
Benchmarks framework isn't working after move to shared framework

### ISSUE THREAD

**Issue by DavidObando** [aspnet/Performance#119]
It appears that our move to a shared runtime broke the benchmark test discoverer or changed the way xUnit tests are picked up. I haven't confirmed this issue is related to the change to the shared framework, I'm just assuming it is as it was working well last week and no additional code changes were checked in.

We're seeing that tests marked with `[Fact]` or `[Theory]` are being picked up, but tests marked as `[Benchmark]` aren't.

This is a blocking bug for the performance tests.

/cc @NTaylorMullen @troydai @yuwaMSFT @sivagms

**Comment by NTaylorMullen** [comment]
Interesting. Does this also occur when you do `dotnet test` at the command line.?

**Comment by DavidObando** [comment]
Here's what I see when I do `dotnet test`
```
D:\GitHub\aspnet\Performance\test\Microsoft.AspNetCore.Tests.Performance [daobando/testfixes]> dotnet test
Project Benchmarks.Framework (.NETStandardApp,Version=v1.5) was previously compiled. Skipping compilation.
Project Benchmarks.Utility (.NETStandardApp,Version=v1.5) was previously compiled. Skipping compilation.
Project Microsoft.AspNetCore.Tests.Performance (.NETStandardApp,Version=v1.5) was previously compiled. Skipping compilation.
Project Benchmarks.Framework (.NETFramework,Version=v4.5.1) was previously compiled. Skipping compilation.
Project Benchmarks.Utility (.NETFramework,Version=v4.5.1) was previously compiled. Skipping compilation.
Project Microsoft.AspNetCore.Tests.Performance (.NETFramework,Version=v4.5.1) was previously compiled. Skipping compilation.
Project Benchmarks.Framework (.NETFramework,Version=v4.5.1) was previously compiled. Skipping compilation.
Project Benchmarks.Utility (.NETFramework,Version=v4.5.1) was previously compiled. Skipping compilation.
Project Microsoft.AspNetCore.Tests.Performance (DNX,Version=v4.5.1) was previously compiled. Skipping compilation.
dotnet-test Error: 0 : Microsoft.DotNet.Cli.Utils.CommandUnknownException: No executable found matching command "dotnet-test-xunit"
   at Microsoft.DotNet.Cli.Utils.ProjectDependenciesCommandFactory.FindProjectDependencyCommands(String commandName, IEnumerable`1 commandArgs, String configuration, NuGetFramework framework, String outputPath, String buildBasePath, String projectDirectory)
   at Microsoft.DotNet.Cli.Utils.ProjectDependenciesCommandFactory.Create(String commandName, IEnumerable`1 args, NuGetFramework framework, String configuration)
   at Microsoft.DotNet.Tools.Test.ConsoleTestRunner.DoRunTests(ProjectContext projectContext, DotnetTestParams dotnetTestParams)
   at Microsoft.DotNet.Tools.Test.TestCommand.DoRun(String[] args)
```

If you want to play with this and have issues building stuff, I recommend you start from the changes I've done in branch [daobando/testfixes](https://github.com/aspnet/Performance/tree/daobando/testfixes).

**Comment by DavidObando** [comment]
@moozzyk is this related to what you mentioned today during standup?

**Comment by moozzyk** [comment]
@DavidObando - I think it's different. In my case runtime could not be found at all because I changed tests and tools to be netcoreapp1.0.

**Comment by SajayAntony** [comment]
Can we close this?

**Comment by DavidObando** [comment]
This is the only standing quality issue I haven't attended. Will take a look now.

**Comment by DavidObando** [comment]
Addressed by #133
